scripts/worldcover/embeddings_db.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+
+import geopandas as gpd
+import lancedb
+import matplotlib.pyplot as plt
+from skimage import io
+
+# Set working directory
+wd = "/home/usr/Desktop/"
```

Comment:
Thanks for the feeback @chuckwondo you are right, this isn't great, but is supposed to be a placeholder. Sometimes I use fake paths like `/path/to/your/working/directory`, to show what this is supposed to be so that people running the script could replace it.

But I am very happy to learn about better ways to do this, what is your favorite solution for this kind of thing? In cases like this with scripts, maybe env vars could be an option?

Reply:
I'm happy to propose some ideas, but I need some context first. How is this script intended to be used? Is the intent to have the user first run the `aws s3 sync` command shown in the code comment below this line, and then just directly call this script?

Reply:
Yes exactly, the idea is that someone with access to the embeddings downloads them using `aws s3 sync` to a local folder, and then runs the script pointing to the embedding files and to a folder where the lancedb data should be stored.

I.e. the script needs two folders

1. A place to download the raw data
2. A folder to create / store the lancedb data

Reply:
Add command-line arguments. For very simple scripts like this one, simply use Python's standard [argparse](https://docs.python.org/3/library/argparse.html) module, so you don't have to add any dependencies. In this case, it sounds like you might want to use `argparse` to support a syntax like the following for running the script:

```
scripts/worldcover/embeddings_db.py --input-dir path/to/input/dir --db-dir path/to/db/dir
```

Both options should be required, and the script should also create the dir specified for `--db-dir`, so the user doesn't have to do so manually beforehand.